packages/@okta/vuepress-site/docs/reference/rl-additional-limits/index.md

```diff
@@ -75,15 +75,53 @@ These rate limits apply to all new Okta organizations. For orgs created before 2
 
+### Okta-generated email message rate limits
+
+Limits are applied on a per-recipient basis and vary by email type. The limit for some email types is no more than 30 emails per-recipient, per-minute, while other email types are configured with higher limits. These limits protect your org against denial-of-service attacks and help ensure that adequate resources are available for all customers.
+
 ### Per-user limits
 
 API endpoints that take username and password credentials, including the [Authentication API](/docs/reference/api/authn/) and the [OAuth 2.0 resource owner password flow](/docs/guides/implement-password/), have a per-username rate limit to prevent brute force attacks with the user's password:
 
-| Action and Okta API Endpoint                                      | Per User Limits (All Orgs) |
+| Action and Okta API endpoint                                      | Per User limits (all orgs) |
 | ----------------------------------------------------------------- | -------------------------: |
 | **Authenticate the same user:**<br>`/api/v1/authn`                | 4 per second               |
 | **Generate or refresh an OAuth 2.0 token:**<br>`/oauth2/v1/token` | 4 per second               |
 
-### Okta-generated email message rate limits
+### Workforce license rate limit multiplier
 
-Limits are applied on a per-recipient basis and vary by email type. The limit for some email types is no more than 30 emails per-recipient, per-minute, while other email types are configured with higher limits. These limits protect your org against denial-of-service attacks and help ensure that adequate resources are available for all customers.
+Workforce orgs that are created after December 17, 2020 have increased default rate limits. This increase is for [specific endpoints](#list-of-endpoints) and depends on a Workforce org's license count (Universal Directory or Single-Sign On).
+
+
+| Workforce licenses | Rate limit multiplier      |
+| ------------------ | -------------------------: |
+| < 10K              | 1x the default rate limit  |
+| 10K - 100K         | 5x the default rate limit  |
+| > 100K             | 10x the default rate limit |
+
+#### List of endpoints
+
+[Authentication](/docs/reference/rl-global-enduser/)
+
+* `/api/v1/authn`
+* `/api/v1/authn/factors/{factorIdOrFactorType}/verify`
+* `/api/v1/sessions`
+* `/login/login.htm`
+* `/login/sso_iwa_auth`
+* `/api/{apiVersion}/radius`
+
+[Authorization](/docs/reference/rl-global-enduser/)
+
+* `/oauth2/{authorizationServerId}/v1`
+* `/oauth2/v1`
+* `/app/{app}/{key}/sso/saml`
+* `/app/office365{appType}/{key}/sso/wsfed/active`
+* `/app/office365{appType}/{key}/sso/wsfed/passive`
+* `/app/template_saml_2_0/{key}/sso/saml`
+
+[Single User/Group/App operations (GET, UPDATE, and DELETE)](/docs/reference/rl-dynamic-scale/)
+
+* `/api/v1/apps/{id}`
+* `/api/v1/groups/{id}`
+* `/api/v1/users/{id}`
+* `/api/v1/users/{idOrLogin}`
```

Comment:
why is `/api/v1/users/me` not included ?

Reply:
I discussed with @karthicksolaimalai-okta and he said: I could remove that endpoint from the list because it was an old endpoint that we don't need to doc externally.